Packs/GCP-IAM/Integrations/GCPIAM/GCPIAM.py

```diff
@@ -3838,11 +3839,12 @@ def gcp_iam_tagbindings_list_command(client: Client, args: Dict[str, Any]) -> Un
     return command_results
 
 
-def test_module(service_account_key: str) -> None:
+def test_module(service_account_key: str, proxy: bool, disable_ssl_certificate: bool) -> None:
     try:
-        client: Client = Client(client_secret=service_account_key)
+        client: Client = Client(client_secret=service_account_key, proxy=proxy, disable_ssl_certificate=disable_ssl_certificate)
```

Comment:
think maybe this is worth a validation? Not the first time we've had this error. Lets think about it and open a ticket if relevant

Reply:
which kind of validation do you have in mind?